tests/Auth0.AuthenticationApi.IntegrationTests/DatabaseConnectionTests.cs

```diff
@@ -75,65 +75,14 @@ public async Task Can_signup_user_and_change_password()
             signupUserResponse.Picture.Should().Be(signupUserRequest.Picture);
         }
 
-        [Fact]
-        public async Task Signup_Response_Normalizes_Id_For_ImportDb()
-        {
-            // Sign up the user
-            var signupUserRequest = new SignupUserRequest
-            {
-                ClientId = GetVariable("AUTH0_CLIENT_ID"),
-                Connection = "integration-importdb",
-                Email = $"{Guid.NewGuid():N}@nonexistingdomain.aaa",
-                Password = Password
-            };
-            var signupUserResponse = await _authenticationApiClient.SignupUserAsync(signupUserRequest);
-            signupUserResponse.Should().NotBeNull();
-            signupUserResponse.Id.Should().NotBeNull();
-            signupUserResponse.Email.Should().Be(signupUserRequest.Email);
-        }
```

Comment:
I removed this test because I dont see what is being tested here that is different from `Signup_Response_Normalizes_Id_For_RegularDb` in terms of our SDK. Sure it's testing another kind of database, but it is testing the same functionality in our SDK. So sounds more like something that is testing Auth0 API's more than testing our SDKs.